src/style.css

```diff
@@ -0,0 +1,88 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 800;
+  font-style: normal;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #687c94;
+}
+
+.big-search-bar {
+  font-family: inherit;
+  border-radius: 4px;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  color: #3d4e61;
+
+  margin: 20px 0 0;
+  display: block;
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
```

Comment:
add font size, now your text is smaller than it should, that's why you have failing tests